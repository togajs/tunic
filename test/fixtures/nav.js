/**
 * @title Navigation Class
 * @name foo
 * @parent bar
 */
