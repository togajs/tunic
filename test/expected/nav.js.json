{
  "type": "Document",
  "blocks": [
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": []
    },
    {
      "type": "Code",
      "contents": ""
    }
  ],
  "nav": {
    "parent": "bar",
    "name": "foo",
    "title": "Navigation Class"
  }
}
